src/Symfony/Bridge/Doctrine/Validator/Constraints/UniqueEntityValidator.php

```diff
@@ -135,13 +136,29 @@ public function validate($entity, Constraint $constraint)
         $errorPath = null !== $constraint->errorPath ? $constraint->errorPath : $fields[0];
         $invalidValue = isset($criteria[$errorPath]) ? $criteria[$errorPath] : $criteria[$fields[0]];
 
+        $vars = array();
+        $paramaters = array();
+
+        if (preg_match_all('/{{ ([a-zA-Z0-9_]+) }}/', $constraint->message, $vars) > 0) {
+
+            if (!class_exists('Symfony\\Component\\PropertyAccess\\PropertyAccess')) {
+                throw new \RuntimeException('Unable to access entity property as the Symfony PropertyAccess is not installed.');
```

Comment:
`PropertyAccess` component should be ship with standard edition right ? Why this check ?